app/web/public/demo-worker.js

```diff
@@ -0,0 +1,625 @@
+(function(f) {
```

Comment:
I understand why you've done it, having this file just hanging in the public folder is not something I'd like to have long term, mostly because unlike the rest of the repo this isn't transpiled so suddenly we have to be concious of old browsers just for this one file.

Reply:
There are 2 ways to run a worker. 
1. js file
2. generated code

option 2. would enable bundling the code together with rest of the app and can be done in two ways
 - paste the whole worker code into a string
 - wrap the worker code in a function and use funtion.toString() to extract the code later

worker is still heavily under construction, and currently this way it is easier to debug for me. After it is further improved we will look to integrate it instead of having it like this in public folder.

Reply:
Fair enough.

Once you're happy with it/it's settled down a little, I'd be happy to work this into the bundle. I did something similar in another webpack project
https://github.com/zalo/CascadeStudio/pull/58/files#diff-d4a06c27ddf0c24f2ba6029bffc9b20062214e07d6ee72822997fb6941b2534d